### PR TITLE
feat: refactor readme example

### PR DESCRIPTION
**Pull request by jordibonet-lambdaclass**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: validium_mode_example/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Run the example
+
+In this example we're going to run some transactions.
+
+Once the server is running, run this command in other terminal:
+```sh
+cargo run --release --bin zksync_full_stack
+```
+> [!IMPORTANT]
+> Make sure that you are in the `validium_mode_example` directory.
```
Comment (toni-calvin) [inline validium_mode_example/README.md:27]: This is not true

File: validium_mode_example/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Transfer 1000
+Transaction hash 0x7cfadf74a5fa571ed9e2f1a115edc62f8db913d61d387177b7b07e2cb270af75
+Transaction gas used 125466
+L2 fee: 31366500000000
+L1 max fee per gas: 1000000010
+```
+
+> [!NOTE]
+> You can observe how the diferent fields evolve depending on the operation. The `transaction hash` is a changing field and the value of the other fields may change depending on the server mode (`rollup` or `validium`).
```
Comment (toni-calvin) [inline validium_mode_example/README.md:71]: I would not put this here

**Review by toni-calvin — APPROVED** [review]
Left some somments, but lgtm!

**Comment by jordibonet-lambdaclass** [comment]
If anyone wants to see how it looks like with the block quotes: https://github.com/lambdaclass/zksync-era/tree/validium-example-readme/validium_mode_example

**Comment by jordibonet-lambdaclass** [comment]
> The PR looks great! We just need to remove the old `validium.md`

Ok! I had thought about putting some information about Validium in that file, but the (official ZKSync page)[https://docs.zksync.io/reference/concepts/validiums.html] is down so you're right I'd better remove the file.

**Review by ilitteri — APPROVED** [review]
LGTM

**Pull request merged**